time stamp
float32 FilteredRange
float32 UnfilteredRange
